lib/config_cat/fetch_policy.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ConfigCat.FetchPolicy do
+  defstruct [
+    :type,
+    cache_expiry_seconds: 0,
+    poll_interval_seconds: 0
+  ]
```

Comment:
We will need an on_configuration_changed_callback parameter here and we will need to call it if the configuration changed.

Reply:
#9